.github/workflows/ci.yml

```diff
@@ -52,6 +49,14 @@ jobs:
           # but not marked as such will fail.
           unshare --map-root-user --net make test TEST_ARGS="--skip-online -vv --showlocals"
 
+      - name: test
+        run: make test TEST_ARGS="-vv --showlocals"
+
+      - uses: ./.github/actions/upload-coverage
+        # only aggregate test coverage over linux-based tests to avoid any OS-specific filesystem information stored in
+        # coverage metadata.
+        if: ${{ matrix.conf.os == 'ubuntu-latest' }}
```

Comment:
Out of curiosity: why did you reorder these test blocks? No problem either way.

Reply:
Makes sense, and sounds good then! Offline tests should always be a subset of the full suite, so this should be fine.